### DISCUSSION TITLE
Mac OS safari can not read spaces,and also other pc can not read safari's space too

### DISCUSSION THREAD

**Discussion by betheboy** [docmost/docmost#481] [Help / Questions]
As what i said, mac os safari can not read spaces create by pc! 
But chrome in mac os can read spaces.
Server version V0.4.1

**Comment by Philipinho** [comment]
Hi, I am not sure I fully understand what you mean by "read spaces". Can you please provide more details and screenshots if possible?

**Comment by betheboy** [comment]
![image](https://github.com/user-attachments/assets/82f67bec-6490-4ee3-baa4-7c4bfc97afdf)
As you can see in this pic.safari can not read spaces. 
In the red circl there had content,but safari can not print it!
